test_discord_mentions.py

```
@@ -0,0 +1,188 @@
+"""
+Test script to verify Discord mentions in summaries.
+This script tests that the LLM receives user mappings and generates proper Discord mentions.
+"""
+
+import asyncio
+import unittest
+from unittest.mock import AsyncMock, MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.AsyncMock` imported but unused

Remove unused import: `unittest.mock.AsyncMock`

(F401)